tests/unit/test_define_xml_reader.py

```diff
@@ -106,11 +106,14 @@ def test_extract_domain_metadata():
         }
 
 
-def test_extract_variable_metadata():
+@pytest.mark.parametrize(
+    "filename", [(test_define_file_path), (test_define_2_0_file_path)]
```

Comment:
you should also add this parametrization to `test_extract_domain_metadata` to verify class and is_non_standard